src/routes.tsx

```
@@ -3,10 +3,16 @@ import DesignController from "./pages/design/controller/index.controller";
 import ProductController from "./pages/product/controller/index.controller";
 import AgilityController from "./pages/agility/controller";
 import TechRecruiterController from './pages/tech-recruiter/controller/index.controller';
+import AgilityController from './pages/agility/controller';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AgilityController'. Consider to delete it or rename it.

'AgilityController' is defined here:


(lint/suspicious/noRedeclare)